osmosis-1/osmosis.zone.json

```diff
@@ -293,6 +293,13 @@
       "osmosis_frontier": true,
       "osmosis_info": true
     },
+    {
+      "chain_name": "uptick",
+      "base_denom": "auptick",
+      "osmosis_main": true,
+      "osmosis_frontier": true,
```

Comment:
frontier tag requires that at least one pool be defined. If needed, I can generate for you a staging link to create a pool.